src/imio/smartweb/common/adapters.zcml

```diff
@@ -7,4 +7,14 @@
       factory=".adapters.BaseCroppingProvider"
       />
 
+  <unconfigure>
+    <adapter name="image_contenttype" factory="plone.namedfile.field.ImageContenttypeValidator" />
+  </unconfigure>
```

Comment:
I tried the adapter without the `unconfigure`, and for me it is working fine : your adapter and the original one are both called.
We need to check that 😉